www/js/controller/report.address.controller.js

```diff
@@ -3,21 +3,25 @@ angular.module('vamosJuntas')
         function($scope, $stateParams, $location, $ionicHistory, reportFactory) {
 
             $scope.report = {};
-            $scope.report.address = $stateParams.address;
-            $scope.report.latitude = $stateParams.latitude;
-            $scope.report.longitude = $stateParams.longitude;
+            $scope.search = {};
+            $scope.search.text = $stateParams.address;
+            $scope.search.latitude = $stateParams.latitude;
+            $scope.search.longitude = $stateParams.longitude;
 
             $scope.myGoBack = function() {
                 $ionicHistory.goBack();
             };
 
             var reportANewRisk = function() {
+                $scope.report.address = $scope.search.text;
+                $scope.report.latitude = $scope.search.latitude;
+                $scope.report.longitude = $scope.search.longitude;
+
                 reportFactory.createReport($scope.report).success(function(data, status, headers, config) {
```

Comment:
On input of report-address, change the name from `text` to `address`, and instead of pass `$scope.report` in createReport function, use directly the `$scope.search` because they will be the same object, let me know if it makes more clear now